app/scripts/models/settings.js

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ * Q-MUNICATE settings Module
+ *
+ */
+define([
+    'jquery',
+    'config'
+], function($, Backbone, QMCONFIG) {
+    'use strict';
+
+    var globalSettings;
+
+    function Settings(app) {
+        this.app = app;
+        this.userSettings = {};
+        globalSettings = this.userSettings;
+        globalSettings.messages_notify = true;
+        globalSettings.calls_notify    = true;
+        globalSettings.sounds_notify   = true;
+    }
+
+    Settings.prototype = {
+
+        // set users settings from localStorage or create default (default - all is ON)
+        setUp: function() {
+            // sets default settings if they are not there in the localStorage
+            if (!localStorage['QM.settings']) {
```

Comment:
If you want merge setting this called sync.
For example, sync with server or sync with storage